requirements/optional.txt

```diff
@@ -1,6 +1,5 @@
 albumentations>=0.3.2
 cityscapesscripts
 imagecorruptions
-mmlvis
```

Comment:
may switch from mmlvis to lvis?

Reply:
lvis-api pypi package is not up-to-date. Detectron2 also installs LVIS from GitHub directly.